### ISSUE TITLE
sitemap.xml not working with php 8.1.13

### ISSUE THREAD

**Issue by dennisbaum** [bnomei/kirby3-feed#53]
If i add your feed-plugin for a basic `sitemap.xml` to a `PHP 8.1.13` with `K3.7.5`, i get the following error regarding `strtotime`.  

```shell
strtotime(): Passing null to parameter #1 ($datetime) of type string is deprecated
```
any idea how to avoid that. maybe add another dateformat to the options?

```js
[
    'dateformat' => 'c',
]
```

Tnx!  

![grafik](https://user-images.githubusercontent.com/562826/208886653-72ab8e8a-80ac-4693-bf59-302551980b29.png)

**Comment by bnomei** [comment]
i can not replicate this.

unless you set the snippet to a custom one it should have `c` by default.
https://github.com/bnomei/kirby3-feed/blob/ae2668a9e037d46e54680b142c257bf8f578b14e/index.php#L24

maybe you have a content where the called field is `null`/empty?
https://github.com/bnomei/kirby3-feed/blob/ae2668a9e037d46e54680b142c257bf8f578b14e/classes/Feed.php#L160

**Comment by bnomei** [comment]
fixed in kirby 3.9.2-rc.1

**Comment by bnomei** [comment]
and this plugin latest release
